ui/lib/kv/addon/components/page/secret/paths.hbs

```diff
@@ -0,0 +1,65 @@
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
+  <:tabLinks>
+    <LinkTo @route="secret.details" data-test-secrets-tab="Secret">Secret</LinkTo>
+    <LinkTo @route="secret.metadata.index" data-test-secrets-tab="Metadata">Metadata</LinkTo>
+    <LinkTo @route="secret.paths" data-test-secrets-tab="Paths">Paths</LinkTo>
+    {{#if @secret.canReadMetadata}}
+      <LinkTo @route="secret.metadata.versions" data-test-secrets-tab="Version History">Version History</LinkTo>
+    {{/if}}
+  </:tabLinks>
+</KvPageHeader>
+
+<h2 class="title is-5 has-top-margin-l">
+  Paths
+</h2>
+
+<div class="box is-fullwidth is-sideless is-paddingless is-marginless">
+  {{#each this.paths as |path|}}
+    <InfoTableRow @label={{path.label}} @labelWidth="is-one-third" @helperText={{path.text}}>
+      {{! replace with Hds::Copy::Snippet }}
+      <CopyButton
+        class="button is-compact is-transparent level-right"
+        @clipboardText={{path.snippet}}
+        @buttonType="button"
+        @success={{action (set-flash-message (concat path.label " copied!"))}}
+      >
+        <Icon @name="clipboard-copy" aria-label="Copy" />
+      </CopyButton>
+      <code class="has-left-margin-s level-left">
+        {{path.snippet}}
+      </code>
+    </InfoTableRow>
+  {{/each}}
+</div>
+
+<h2 class="title is-5 has-top-margin-l">
+  Commands
+</h2>
+
+<div class="box is-fullwidth is-sideless">
+  <h3 class="is-label">
+    CLI
+    <Hds::Badge @text="kv get" @color="neutral" />
+  </h3>
+  <p class="helper-text has-text-grey has-bottom-padding-s">
+    This command retrieves the value from K/V secrets engine at the given key name. For other CLI commands,
```

Comment:
Maybe this came from design, but in the [docs](https://developer.hashicorp.com/vault/docs/secrets/kv) I've seen it's always referred to as KV secrets engine (not K/V)?

Reply:
Uh... the docs links it points to has it as k/v... and K/V. I honestly think we should change those docs? https://developer.hashicorp.com/vault/docs/commands/kv